credential-viewer-component.js

```diff
@@ -141,13 +141,20 @@ function Ctrl(
   };
 
   self.confirmDeleteCredential = function(err, result) {
+    var credentialServiceOptions = {};
+    if(!compareHost(self.credential.id)) {
+      credentialServiceOptions.url = self.altUpdateEndpoint + '?id=' +
+        encodeURIComponent(self.credential.id);
+    }
```

Comment:
I'd prefer for us to get rid of this magic somehow, but don't know how to yet, so it's fine for now.